frontend/src/static/js/v_summary.js

```diff
@@ -282,6 +301,32 @@ window.vSummary = {
       this.getDates();
       this.sortFiltered();
     },
+    processFileTypes() {
+      const selectedColors = ['#e6194b', '#3cb44b', '#ffe119', '#4363d8', '#f58231', '#911eb4', '#46f0f0', '#f032e6',
+        '#bcf60c', '#fabebe', '#008080', '#e6beff', '#9a6324', '#fffac8', '#800000', '#aaffc3', '#808000', '#ffd8b1',
+        '#000075', '#808080'];
+
+      this.repos.forEach((repo) => {
+        const user = repo.users[0];
+        const colors = {};
+        let i = 0;
+        if (Object.keys(user.fileTypeContribution).length === 15) {
+          user.fileTypeContribution = this.sortKeysAlphabetically(user.fileTypeContribution);
```

Comment:
why is it only sorted if its === 15?

Reply:
It was meant to check if the file formats are the default file formats, as it should only be sorted alphabetically if they are the default file formats, otherwise ordering should remain the same. I have replaced this with a ```isDefaultFileFormats``` method for better checking